src/main/java/se/sundsvall/supportmanagement/api/NotificationsResource.java

```diff
@@ -74,6 +76,19 @@ ResponseEntity<Notification> getNotification(
 		return ok(notificationService.getNotification(municipalityId, namespace, errandId, notificationId));
 	}
 
+	@GetMapping("/errands/{errandId}/notifications")
+	@ApiResponse(responseCode = "200", description = "Successful operation", useReturnTypeSchema = true)
+	@ApiResponse(responseCode = "404", description = "Not found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	@Operation(summary = "Get notifications", description = "Get a specific notification for the namespace and municipality")
```

Comment:
Description doesnt match the resource. seems to be for
`/errands/{errandId}/notifications/{notificationId}`?